src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/eventDetailsBuilder.hpp

```diff
@@ -229,6 +231,18 @@ class TEventDetailsBuilder final : public AbstractHandler<std::shared_ptr<TScanC
                 ecsData["vulnerability"]["published_at"] = returnData.data->datePublished()->str();
                 ecsData["vulnerability"]["reference"] = returnData.data->reference()->str();
                 ecsData["vulnerability"]["scanner"]["vendor"] = "Wazuh";
+                try
+                {
+                    ecsData["vulnerability"]["scanner"]["source"] = TGlobalData::instance()
+                                                                        .vendorMaps()
+                                                                        .at("adp_descriptions")
+                                                                        .at(data->m_vulnerabilitySource.first)
+                                                                        .at("adp");
+                }
+                catch (const std::exception& e)
+                {
+                    ecsData["vulnerability"]["scanner"]["source"] = " ";
```

Comment:
To align with this issue:
- https://github.com/wazuh/wazuh/issues/25484

```suggestion
                    ecsData["vulnerability"]["scanner"]["source"] = "-";
```